support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -34,6 +35,7 @@ object SendThankYouEmailState {
       paymentMethod: PaymentMethod,
       accountNumber: String,
       subscriptionNumber: String,
+      abTests: Option[Set[AbTest]],
```

Comment:
We need to know whether the user who has created a sub is in the ab test variant so that we can tell which version they have purchased.